### DISCUSSION TITLE
How to use the work in progress packages?

### DISCUSSION THREAD

**Discussion by ambosch** [microsoft/python-type-stubs#277] [Q&A]
Hi,
I love the work that you do and would like to make use of the type stubs you provide. Specifically, I want to use the sklearn stubs package. How can I make use of these stubs in my own project? I use poetry for package management. From my understanding, it it still in development and not published to install/add via poetry? What is you advice? Looking forward to hearing from you!

**Comment by debonte** [comment]
@ambosch, what type checker are you using? You may find https://github.com/microsoft/python-type-stubs/issues/264 useful. Please chime in there if you have anything to add.
